core/chain_manager.go

```diff
@@ -341,13 +341,24 @@ func (bc *ChainManager) ResetWithGenesisBlock(gb *types.Block) {
 
 // Export writes the active chain to the given writer.
 func (self *ChainManager) Export(w io.Writer) error {
+	if err := self.ExportN(w, uint64(1), self.currentBlock.NumberU64()); err != nil {
```

Comment:
This will fail if the chain is zero length. since 1 > 0 
I guess the second param of ExportN should be 0 anyway and that shouldnt fail since 0 is the genesis block.